src/components/ConnectButton/ConnectButton.tsx

```diff
@@ -5,24 +5,29 @@ import { useEthereum } from '@/contexts/EthereumContext';
  * Ethereum-based blockchain. It leverages the useEthereum hook
  * from the EthereumContext for managing blockchain connections.
  */
-const ConnectButton: React.FC = () => {
-  const { connect, disconnect, account } = useEthereum();
 
+interface Props {
+  connectText: string;
+  disconnectText: string;
+}
```

Comment:
im not even sure this data should come from the parent via props. Right now this button is used only at one place.